### ISSUE TITLE
Upgrade to Kubernetes 1.9.6

### ISSUE THREAD

**Issue by zegl** [kubernetes/minikube#2651]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

FEATURE REQUEST

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`): v0.25.2

**What happened**:

1.9.4 contains a regression causing `subPath` for volume mounts to not work correctly. It has been fixed in https://github.com/kubernetes/kubernetes/pull/61080 and was shipped in 1.9.5.

**What you expected to happen**:

Minikube to use Kubernetes 1.9.6.

What is the current timeline for upgrading to 1.9.6?

**Comment by rmorrise** [comment]
This is a show-stopper bug. I had to recreate my minikube VM and downgrade to Kube 1.9.0 to work around this issue. (minikube windows x64)

**Comment by rmorrise** [comment]
Duplicate of: https://github.com/kubernetes/minikube/issues/2642

**Comment by MichaelScript** [comment]
This is now fixed in minikube v0.26.0 which you can update to on OSX with:
```
curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.26.0/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```
https://github.com/kubernetes/minikube/releases

It doesn't look like it's using 1.9.6 but subpath now works.

**Comment by zegl** [comment]
@MichaelScript Nice!
